Automattic/wp-calypso
# Segmentation Survey: Integrate survey component with entrepreneur flow

Closes https://github.com/Automattic/wp-calypso/issues/89550

## Proposed Changes

* Integrate the Segmentation Survey component with the entrepreneur flow
* Remove "e-commerce segmentation flow", which was created for testing purposes
* Update navigation logic to use hash parameters
* Hide back button on the first page

## Testing Instructions

* Apply this PR to your local
* Go to http://calypso.localhost:3000/setup/entrepreneur/start?flags=ecommerce-segmentation-survey
* Check if the survey is properly displayed
* Navigate through the questions and verify if the save request is performed
  * The hash param should be updated in the URL
* After the last question, the step is submitted, and the site created (the flow development is still in progress)

## Pre-merge Checklist
